package.json

```diff
@@ -2121,6 +2135,15 @@
               }
             }
           }
+        },
+        "windows": {
+          "program": "./.debugger/vsdbg-ui.exe"
+        },
+        "osx": {
+          "program": "./.debugger/vsdbg-ui"
+        },
+        "linux": {
+          "program": "./.debugger/vsdbg-ui"
```

Comment:
I'm pretty sure that you don't want these changes in your PR. Is that correct @gregg-miskelly?

Reply:
Correct

Reply:
@janaka: You need to revert this bit.